keyboards/dmqdesign/spin/keymaps/encoderLayers/keymap.c

```diff
@@ -0,0 +1,151 @@
+/* Copyright 2019-2020 DMQ Design
```

Comment:
Could you please rename this keymap to lowercase?

Reply:
Renamed the keymap to lowercase, had to do some funky stuff since I'm new to anything past the base github GUI but should be fixed.